plugin/controller/lib/impl/mcp/MCPControllerRegister.ts

```
@@ -70,13 +74,105 @@
     this.controllerMeta = controllerMeta;
+
+    this.mcpConfig = new MCPConfig(app.config.mcp);
   }
 
   static addHook(hook: MCPControllerHook) {
     MCPControllerRegister.hooks.push(hook);
   }
 
-  async mcpStreamServerInit() {
+  static async connectStatelessStreamTransport() {
+    if (MCPControllerRegister.instance && MCPControllerRegister.instance.statelessTransport) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)